lib/headlines/security_headers/x_frame_options.rb

```diff
@@ -15,7 +15,7 @@ def score_by_value
       end
 
       def score_by_allow_from
-        whitelisted_domain? ? 1 : 0
+        whitelisted_domain? ? 1 : -1
```

Comment:
It's possible I updated scoring elsewhere without updating the scoring spreadsheet, but in this particular case I haven't touched the file (https://github.com/srcclr/security-headers/commits/master/lib/headlines/security_headers/x_frame_options.rb).